en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md

```diff
@@ -34,6 +34,96 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
                 apictl init SampleAPI --definition definition.yaml --force=true                
                 ```
 
+            !!! note
+                Note that API definition is different from Swagger2 or OpenAPI3 specification. The following is a sample API definition used to generate an API Project.
```

Comment:
Lets make this note more precise. We can say,

```
Note that the API definition can either be a Swagger 2 or OpenAPI 3 specification. The following is a sample <Swagger 2 or OpenAPI 3> specification used to generate an API Project.
```

Reply:
Since the definition is different from the Swagger 2 and OpenAPI specification, the above description is correct. But that description might be confusing users, so let's change the description via https://github.com/wso2/docs-apim/pull/2559/commits/708589ec269c76201da4254ec4d018824fa1cfd8